microcosm_postgres/encryption/encryptor.py

```diff
@@ -5,16 +5,16 @@
 from typing import Mapping, Sequence, Tuple, Union
 
 from aws_encryption_sdk import decrypt, encrypt
-from aws_encryption_sdk.key_providers.base import MasterKeyProvider
+from aws_encryption_sdk.materials_managers.base import CryptoMaterialsManager
 
 
 class SingleTenantEncryptor:
     """
     A single tenant encryptor.
 
     """
-    def __init__(self, key_provider: MasterKeyProvider):
-        self.key_provider = key_provider
+    def __init__(self, materials_manager: CryptoMaterialsManager):
```

Comment:
No real change here, we will use by default `DefaultCryptoMaterialsManager` that is (implicit) used today
ref: https://github.com/aws/aws-encryption-sdk-python/blob/b3bf59c0fd9b6a6ef46f0f94476f5db38e8bedb9/src/aws_encryption_sdk/streaming_client.py#L99